content/webapp/components/EventDatesLink/EventDatesLink.tsx

```diff
@@ -3,28 +3,33 @@ import { font } from '@weco/common/utils/classnames';
 import { trackEvent } from '@weco/common/utils/ga';
 import Icon from '@weco/common/views/components/Icon/Icon';
 import { arrowSmall } from '@weco/common/icons';
+import styled from 'styled-components';
 
 type Props = {
   id: string;
 };
 
-const EventDatesLink: FunctionComponent<Props> = ({ id }: Props) => {
-  return (
-    <a
-      href="#dates"
-      onClick={() => {
-        trackEvent({
-          category: 'EventDatesLink',
-          action: 'follow link',
-          label: id,
-        });
-      }}
-      className={`flex-inline flex-v-center ${font('intb', 5)}`}
```

Comment:
Note: there is no class `flex-v-center`, it's a typo. I've applied the style anyway, but if stuff goes wonky then this class will have sprung into life.